### PR TITLE
[GEOS-9186] Allow logging filter to also dump request headers

### PR DESCRIPTION
**Pull request by aaime**

### CODE REVIEW

**Comment by aaime** [comment]
Tried, works:

image

File: src/main/src/main/java/org/geoserver/filters/LoggingFilter.java [hunk 1]
```diff
@@ -94,6 +105,7 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
     public void init(FilterConfig filterConfig) {
         enabled = getConfigBool("enabled", filterConfig);
         logBodies = getConfigBool("log-request-bodies", filterConfig);
+        logHeaders = getConfigBool("log-request-headers", filterConfig);
```
Comment (jodygarnett) [inline src/main/src/main/java/org/geoserver/filters/LoggingFilter.java]: Okay this is easier to manage next to "log-request-bodies" setting.

I initially wondered why this was not being handled as a log4j configuration option.

**Comment by bradh** [comment]
@aaime Do you still need this branch?

**Comment by aaime** [comment]
Nope

**Pull request merged**